src/Repository/RequestLoggerContext.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+using Repository.Models;
+
+
+namespace Repository;
+
+public class RequestLoggerContext : DbContext
+{
+    /// <summary>
+    /// Context class for entity framework
+    /// </summary>
+    public RequestLoggerContext()
```

Comment:
Is this needed?

Reply:
I think it was initially required for some of the ef migrations, that's now changed, so I've removed it